go.mod

```diff
@@ -3,3 +3,5 @@ module github.com/project-copacetic/scanner-parser-template
 go 1.21
```

Comment:
at [line 1:](https://github.com/project-copacetic/scanner-plugin-template/pull/3/files#diff-33ef32bf6c23acb95f5902d7097b7a1d5128ca061167ec0716715b0b9eeaa5f6R1)
`github.com/project-copacetic/scanner-parser-template` should be `github.com/project-copacetic/scanner-plugin-template` no?

Reply:
thanks, i renamed the repo. will update